gpiozero/output_devices.py

```diff
@@ -797,20 +797,31 @@ class Motor(SourceMixin, CompositeDevice):
         The GPIO pin that the backward input of the motor driver chip is
         connected to.
 
+    :param int enable:
+        (Optional) The GPIO pin that enables the motor. Required for some motor
+        controller boards. Defaults to ``None``.
+
     :param bool pwm:
         If ``True`` (the default), construct :class:`PWMOutputDevice`
         instances for the motor controller pins, allowing both direction and
         variable speed control. If ``False``, construct
         :class:`DigitalOutputDevice` instances, allowing only direction
         control.
     """
-    def __init__(self, forward=None, backward=None, pwm=True):
+    def __init__(self, forward=None, backward=None, enable=None, pwm=True):
         if not all([forward, backward]):
             raise GPIOPinMissing(
                 'forward and backward pins must be provided'
             )
         PinClass = PWMOutputDevice if pwm else DigitalOutputDevice
-        super(Motor, self).__init__(
+        if enable:
```

Comment:
I think this would be better as `if enable is not None:` ?
(since in theory there's nothing stopping you using GPIO 0)